src/components/backend-ai-dialog.ts

```diff
@@ -149,7 +149,12 @@ export default class BackendAiDialog extends LitElement {
    * Hide a dialog.
    */
   _hideDialog() {
-    this.hide();
+    if (this.dialog.parentNode.host.id === 'terms-of-service-dialog') {
```

Comment:
We already have a custom event trigger for the "closed" event in `backend-ai-dialog` component. Please refer to [this code](https://github.com/lablup/backend.ai-webui/blob/dd3e7aa1a5041773dd524ad2d4242d4f68c04f07/src/components/backend-ai-dialog.ts#L136-L141) and implement inside `lablup-terms-of-service`, instead of adding conditional codes in `backend-ai-dialog`.

Reply:
I remove tos-dialog-closing customEvent. Instead, I use `dialog-closed` event to invoke close() function inside `lablup-terms-of-service`